### ISSUE TITLE
Having an options error

### ISSUE THREAD

**Issue by steve-goodwin** [tetranz/select2entity-bundle#5]
Followed all the install instructions and i'm receiving the following error when loading my form:

```
The options "class", "minimum_input_length", "multiple", "page_limit", "placeholder", "remote_route", "text_property" do not exist. Known options are: "action", "attr", "auto_initialize", "block_name", "by_reference", "cascade_validation", "compound", "constraints", "csrf_field_name", "csrf_message", "csrf_protection", "csrf_provider", "csrf_token_id", "csrf_token_manager", "data", "data_class", "disabled", "empty_data", "error_bubbling", "error_mapping", "extra_fields_message", "inherit_data", "intention", "invalid_message", "invalid_message_parameters", "label", "label_attr", "mapped", "max_length", "method", "pattern", "post_max_size_message", "property_path", "read_only", "required", "translation_domain", "trim", "validation_groups", "virtual" 
```

Any advice on this?

Symfony version 2.5.10
Debug info:

CRITICAL - Uncaught PHP Exception Symfony\Component\OptionsResolver\Exception\InvalidOptionsException: "The options "class", "minimum_input_length", "multiple", "page_limit", "placeholder", "remote_route", "text_property" do not exist. Known options are: "action", "attr", "auto_initialize", "block_name", "by_reference", "cascade_validation", "compound", "constraints", "csrf_field_name", "csrf_message", "csrf_protection", "csrf_provider", "csrf_token_id", "csrf_token_manager", "data", "data_class", "disabled", "empty_data", "error_bubbling", "error_mapping", "extra_fields_message", "inherit_data", "intention", "invalid_message", "invalid_message_parameters", "label", "label_attr", "mapped", "max_length", "method", "pattern", "post_max_size_message", "property_path", "read_only", "required", "translation_domain", "trim", "validation_groups", "virtual"" at /Users/Steve/Sites/NavioHealth/patient-app-web/vendor/symfony/symfony/src/Symfony/Component/OptionsResolver/OptionsResolver.php line 260
Context: {"exception":"Object(Symfony\\Component\\OptionsResolver\\Exception\\InvalidOptionsException)"}

**Comment by tetranz** [comment]
Do you have the type set correctly to tetranz_select2entity for this field when you add it to the form with $builder->add()?  Those known options look like what you get for one of the standard field types such as text.

**Comment by steve-goodwin** [comment]
Thanks for you reply, yes the type is set correctly:

$builder
           ->add('receiver_physician', 'tetranz_select2entity', array(
               'multiple' => false,
               'remote_route' => 'ajax_physician_search_by_term',
               'class' => '\HospitalBundle\Entity\Physician',
               'text_property' => 'name',
               'minimum_input_length' => 2,
               'page_limit' => 10,
               'placeholder' => 'Select a Physician',
           ))

**Comment by steve-goodwin** [comment]
Tested on version 2.7.5 with no issues, but 2.5.10 is still erroring out with the same error. Any idea how to fix this? I cannot upgrade the symfony version due to team / release constraints

**Comment by tetranz** [comment]
I'll try to have a play with this on Symfony 2.5.10 over the weekend.

**Comment by tetranz** [comment]
This is fixed.  You'll need to change the version in your composer.json to dev-master. If it works well then I should probably tag a v2.1.

It was caused by the change mentioned here http://symfony.com/blog/new-in-symfony-2-7-form-and-validator-updates .  setDefaultOptions was changed to configureOptions in Symfony 2.7.

@ismailbaskin changed it, correctly, in Select2EntityType with the update to Select2 version 4 but that broke earlier versions of Symfony. I've added setDefaultOptions back to simply call configureOptions.
